### PR TITLE
feat[addd liked videos listing page]

### PR DESCRIPTION
**Pull request by Radheshyam1234**
liked videos

### CODE REVIEW

**Review by HrishikeshMahalle — APPROVED** [review]
The Preview Looks good! Good work

File: src/Components/utils/useUserActions.js [hunk 1]
```diff
@@ -46,6 +49,9 @@ export const useUserActions = () => {
 
       if (status == 200 || 201) {
         dispatch({ type: "SET_LIKED_VIDEOS", payload: likes });
+        let data = JSON.parse(localStorage.getItem("data"));
+        data = { ...data, likes };
```
Comment (tushartiwari7) [inline src/Components/utils/useUserActions.js]: You should try to not mutate the `data` here.

File: src/App.js [hunk 2]
```diff
@@ -3,27 +3,36 @@ import { Routes, Route, useNavigate } from "react-router-dom";
...
   const { state, dispatch } = useStateContext();
+  const { setUserProfile } = useAuthProvider();
   const token = localStorage.getItem("token");
 
   useEffect(() => {
     getVideos(setVideosList);
+    getLikedVideos(dispatch, token);
+    getUserProfile(setUserProfile, token);
     if (token) {
       getLikedVideos(state, dispatch);
```
Comment (tushartiwari7) [inline src/App.js]: Suggestion: while naming a function you should use the `get` suffix only if it's returning something.
But here these functions don't seem to be pure, so you may name it something else.

**Review by tushartiwari7 — APPROVED** [review]
Code looks very clean and understandable. just have a look at my comments once.

**Pull request merged**